test/Sema/immutability.swift

```diff
@@ -61,6 +61,12 @@ class FooClass {
   mutating            // expected-error {{'mutating' isn't valid on methods in classes or class-bound protocols}} {{3-12=}}
   func baz() {}
 
+  nonmutating         // expected-error {{'nonmutating' isn't valid on methods in classes or class-bound protocols}} {{3-15=}}
```

Comment:
How is this source compatibility breaking? Unless I'm missing something, this just improves the QoI of existing errors.